l10n_it_asset_management/__manifest__.py

```diff
@@ -43,6 +43,11 @@
         "wizard/asset_journal_report_view.xml",
         "wizard/asset_previsional_report_view.xml",
     ],
+    "external_dependencies": {
```

Comment:
Se `openupgradelib` serve solo per la migrazione (in `migrations`), non è necessario aggiungerlo qui, vedi https://github.com/OCA/l10n-italy/pull/3343#discussion_r1263496042

Reply:
nei requirements lascio lo stesso `openupgradelib` o pre-commit me l'ha aggiunto solo a causa dell'external dependencies nel manifest ed ora non serve più?

Reply:
> pre-commit me l'ha aggiunto solo a causa dell'external dependencies nel manifest ed ora non serve più

Reply:
:+1: